Authenticator/Source/OTPAppDelegate.swift

```diff
@@ -68,11 +68,11 @@ class OTPAppDelegate: UIResponder, UIApplicationDelegate {
             let alert = UIAlertController(title: "Add Token", message: message, preferredStyle: .Alert)
 
             alert.addAction(UIAlertAction(title: "Cancel", style: .Cancel, handler: nil))
-            alert.addAction(UIAlertAction(title: "OK", style: .Default, handler: { (action) in
-                tokenManager.addToken(token)
+            alert.addAction(UIAlertAction(title: "OK", style: .Default, handler: { [weak self] (action) in
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 106 characters (line_length)